MatrixSDK/Data/Store/MXNoStore/MXNoStore.m

```diff
@@ -433,6 +433,14 @@ - (void)storeHomeserverCapabilities:(MXCapabilities *)homeserverCapabilities
 {
 }
 
+- (MXMatrixVersions *)supportedMatrixVersions
+{
+    return nil;
+}
```

Comment:
```suggestion
}

```

Reply:
Thanks, i won't commit this, as i would like to keep those two methods close.